lbrynet/lbrynet_daemon/auth/auth.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+from zope.interface import implements, implementer
+from twisted.cred import portal, checkers, credentials, error as cred_error
+from twisted.internet import defer
+from twisted.web import resource
+from lbrynet.lbrynet_daemon.auth.util import load_api_keys, APIKey, API_KEY_NAME, save_api_keys
+from lbrynet.lbrynet_daemon.LBRYDaemon import log_dir as DATA_DIR
+
+log = logging.getLogger(__name__)
+
+
+# initialize api key if none exist
+if not os.path.isfile(os.path.join(DATA_DIR, ".api_keys")):
+    keys = {}
+    api_key = APIKey.new()
```

Comment:
cant you pass `name` as a param here instead of calling rename on the next line?